inference-engine/src/low_precision_transformations/src/markup_per_tensor_quantization.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/markup_per_tensor_quantization.hpp"
+
+#include <cassert>
+#include <memory>
+#include <vector>
+#include <ngraph/node.hpp>
+#include "low_precision/rt_info/per_tensor_quantization_attribute.hpp"
+
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::low_precision::MarkupPerTensorQuantization, "MarkupPerTensorQuantization", 0);
+
+ngraph::pass::low_precision::MarkupPerTensorQuantization::MarkupPerTensorQuantization(
+    const std::vector<OperationPerTensorQuantizationRestriction>& restrictions) {
+    for (const OperationPerTensorQuantizationRestriction& restriction : restrictions) {
+        const auto it = restrictionsByOperation.find(restriction.operationType.name);
```

Comment:
You need to work with `type_info` but not with name or version separately.

Reply:
We explicitly separate `type_info::name` and `type_info::version` to be able to work with type name separately (if `specifyVersion` = false).